### ISSUE TITLE
Problem trying to play tv shows from the calendar

### ISSUE THREAD

**Issue by EnderTheDestroyer** [team-crew/team-crew.github.io#33]
Went to TV Shows, Calendar, clicked on Feb 05-Friday (tonight), after the list of shows came up clicked on Gold Rush 11x15-The Viking Detective and the search came back with 36 sources, only one of which even tried to play, Plockers|Mstream. It didn't play. Tried again, clicking on other sources on them list, the same thing happened over and over no matter which link I picked. It's like it's stuck on Plockers|Mstream and can't see any of the other sources.

Good addon, though, the only other issue I see with The Crew isn't your fault, going through the iptv lists looking for a working channel like History or Discover (which I searched for tonight) is tedious. I only found two working sources for Discovery and both stopped playing after a minute or two. I wish whoever made some of the link lists alphabetized them to make searches quicker and checked them periodically to make sure the channels still worked. That's out of The Crew's control though, just wanted to get it off my chest, lol.

Thanks for your time and talent, been using The Crew since it first came out, hope you guys keep it going forever.

**Comment by fade2gray** [comment]
You will rarely see The Crew devs respond in here. The best way to contact them is through the [Telegram](https://telegram.org/) messaging app

**Comment by EnderTheDestroyer** [comment]
Oh well, I don't use Telegram but thanks for letting me know. Do you know if they monitor the reddit addons4kodi sub?

**Comment by fade2gray** [comment]
The Crew isn't an 'approved' Kodi add-on and it is recommended that Telegram is used for contact - can't say much more than that, sorry.

**Comment by EnderTheDestroyer** [comment]
Just so you know, the a4k sub is specifically for unapproved of Kodi addons. People post questions, etc. about The Crew there on a regular basis. I'm not doing Telegram so if they don't monitor github or the a4k sub I guess they'll never know there's a problem.

**Comment by fade2gray** [comment]
"I'm not doing Telegram.." - that's your prerogative but the last response in here from thecrewwh (White Hat) was Oct last year, do they lurk in a4k? - I don't know.

**Comment by EnderTheDestroyer** [comment]
Thanks anyway.

**Comment by thecrewwh** [comment]
If you are using the calendar to try to play stuff on the day it is released no wonder you won't find links that goes with most shows try the day after it is released and see if there are more sources

**Comment by EnderTheDestroyer** [comment]
Nope, same exact problem happens. Clicked on Calendar, then last nights Gold Rush Growing Pains episode, got 39 sources but the only one that tries to play (but doesn't) is plockers|mstream. This has been happening for a while. Fortunately I was able to watch it last night from the calendar with Tempest. This happened last time I tried Gold Rush on The Crew but something's still messed up, I tried watching last nights UnXplained with William Shatner, supposed to be an episode bout gold but an episode about UFOs played when I clicked on cartoonhd|abcvideo source.
